jsonschema-core.md

```diff
@@ -400,16 +400,16 @@ tokens:
 
 - individual Unicode characters, as defined by the [JSON
   specification](#rfc8259);
-- simple character classes ("[abc]"), range character classes ("[a-z]");
-- complemented simple character classes ("[^abc]"),
-  complemented range character classes ("[^a-z]");
-- simple quantifiers: "." (any character except line terminator),
-  "+" (one or more), "*" (zero or more), "?" (zero or one),
-  and their lazy versions ("+?", "*?", "??");
-- range quantifiers: "{x}" (exactly x occurrences), "{x,y}" (at least x, at most
+- simple atoms: `.` (any character except line terminator);
+- simple character classes (`[abc]`), range character classes (`[a-z]`);
+- complemented simple character classes (`[^abc]`),
+  complemented range character classes (`[^a-z]`);
```

Comment:
This feels weird to me.  Either make them separate bullets

```suggestion
- complemented simple character classes (`[^abc]`);
- complemented range character classes (`[^a-z]`);
```

or combine them

```suggestion
- complemented simple (`[^abc]`) and range (`[^a-z]`) character classes (`[^a-z]`);
```

Reply:
I have made the first suggestion; keeping the multi-word terms together before the example makes it easier to search for those terms. I.e, a (cross-)reference to the term "complemented range" is easier to find that way.
See commit ac0a13c